test/features/caseList.feature

```diff
@@ -5,6 +5,10 @@ Feature: Set of Scenarios testing the functionality of the Case List table
     Given I have logged in
 
   Scenario: A Case Reference column in case list results has hyphens every 4 digits
+    Given the definition sheet 'CaseEventToFields' looks like this
+      | CaseTypeID    | CaseEventID   | CaseFieldID          | DisplayContext  |
```

Comment:
I'd use the following formatting:
`| case type id| case event id| case field id| display context|`